spot_rl_experiments/configs/config.yaml

```diff
@@ -58,9 +58,6 @@ WEIGHTS:
   MRCNN_50: "weights/ikea_apricot_r50_normal_100_output_model_0003599.pth"
   DEBLURGAN: "weights/fpn_inception.h5"
   SAM: "weights/sam_vit_h_4b8939.pth"
-  
-# CG Json file
-CONCEPT_GRAPH_FILE : "cfslam_object_relations.json"
```

Comment:
is this replaced by `CG_ROOT_PATH` in `cg_config.yaml`?

Reply:
yes

Reply:
can you modify this function to use `CG_ROOT_PATH` from the new config: https://github.com/facebookresearch/spot-sim2real/blob/b87520f00baf178aadd8057d8f7219735a22e28d/spot_rl_experiments/spot_rl/utils/utils.py#L166